src/drivers/px4flow/px4flow.cpp

```diff
@@ -754,6 +754,10 @@ start(int argc, char *argv[])
 #ifdef PX4_I2C_BUS_ONBOARD
 			PX4_I2C_BUS_ONBOARD,
 #endif
+#ifdef CONFIG_ARCH_BOARD_AEROFC_V1
```

Comment:
Please don't use board specific defines outside of the board support (currently src/drivers/boards/aerofc-v1). 

What about `ifdef PX4_I2C_BUS_EXPANSION1`?